clar2wasm/tests/wasm-generation/main.rs

```diff
@@ -381,6 +382,63 @@ fn qualified_principal() -> impl Strategy<Value = Value> {
     })
 }
 
+fn value_to_string(value: &PropValue, to_response: bool) -> String {
+    match value.0 {
+        Value::Response(_) => format!("{}", value),
+        _ if to_response => format!("(ok {value})"),
+        _ => format!("{}", value),
+    }
+}
+
+fn ensure_last_response(last_value: PropValue, is_public_function: bool) -> PropValue {
```

Comment:
Imagine being someone who didn't write this function. You read it's name, and the function tells you that it ensures that the last expression is a response.

1. Why `last`? There is only one argument here that is a Value, why should we specify it's the last of something?
2. If the second argument is false, the function does nothing! You didn't ensure anything here.
3. I actually had the idea before reading this function that `true` would create a `ok` value and `false` an `err`, but not at all. There is no way to create an `err`.

Reply:
Good point. Will fix it